draft-ietf-quic-transport.md

```diff
@@ -410,6 +414,12 @@ Payload:
 : Octets from 17 onwards (the rest of QUIC packet) are the payload of the
   packet.
 
+FNV Hash:
+
+: If Packet Type is unprotected ( Version Negociation, Client Initial,
+  Client/Server Cleartext, Public Reset ) a 64-bit FNV-1a hash is present.
```

Comment:
You should also update the language on the STREAM frame description of D flag.

Presently, it says: "When set to 0, this field indicates that the Stream Data field extends to the end of the packet.".  

You may want to clarify that in case of a plaintext packet, D=0 indicates that Stream Data field extends up to the FVN-1a hash field.